libs/core/src/global/styles/tokens/tokens.json

```diff
@@ -0,0 +1,437 @@
+{
+  "sage": {
+    "color": {
+      "white": {
+        "white": {
+          "value": "#ffffff",
```

Comment:
Seems all the other color values are uppercase, should this one be as well?

Reply:
Good callout, I will set them all to uppercase.